users/admin.py

```diff
@@ -23,7 +35,17 @@ class ProfileAdmin(admin.StackedInline):
 @admin.register(User)
 class CustomUserAdmin(DjangoUserAdmin):
     inlines: ClassVar[list] = [UserSettingAdmin, ProfileAdmin]
-    list_display: tuple[str] = ("nickname", "email", "is_staff", "is_online", "last_activity_ago")
+    list_display: tuple[str] = (
+        "nickname",
+        "email",
+        "is_staff",
+        "is_online",
+        "last_activity_ago",
+        "is_active",
+        "deactivated_at",
+        "reactivate_until",
+        "reactivate_user_link",
```

Comment:
Please remove `reactivate_user_link ` from list display. This link needs to be visible on user edit page, not on users list.  It can be done as a button pretty easily in user-change-form.html. 
https://github.com/fabiocaccamo/django-admin-interface/discussions/262
And official docs: https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#overriding-admin-templates